runtime/libpgmath/tools/mth_mask.awk

```diff
@@ -160,6 +160,29 @@ function init_target_arrays()
       mask_all_zero = \
         "(_mm512_test_epi32_mask((__m512i)mask, _mm512_set1_epi32(-1)) == 0)"
     }
+  } else if (TARGET == "ARM64") {
+      _mm = "_mm"
+      __m = "__m128"
+
+    divsd["fs"] = _mm "_div_ps((" __m ")x, (" __m ")y)"
+    divsd["fd"] = _mm "_div_pd((" __m "d)x, (" __m "d)y)"
+    divsd["rs"] = _mm "_div_ps((" __m ")x, (" __m ")y)"
+    divsd["rd"] = _mm "_div_pd((" __m "d)x, (" __m "d)y)"
+    divsd["rs"] = _mm "_mul_ps((" __m ")x, " _mm "_div_ps(Csp1_" VL("s") ", ( " __m ")y))"
+    divsd["rd"] = _mm "_mul_pd((" __m "d)x, " _mm "_div_pd(Cdp1_" VL("d") ", ( " __m "d)y))"
+    divsd["ps"] = _mm "_div_ps((" __m ")x, (" __m ")y)"
+    divsd["pd"] = _mm "_div_pd((" __m "d)x, (" __m "d)y)"
+
+    sqrtsd["fs"] = _mm "_sqrt_ps((" __m ")x)"
+    sqrtsd["fd"] = _mm "_sqrt_pd((" __m "d)x)"
+    sqrtsd["rs"] = _mm "_sqrt_ps((" __m ")x)"
+    sqrtsd["rd"] = _mm "_sqrt_pd((" __m "d)x)"
+    sqrtsd["ps"] = _mm "_sqrt_ps((" __m ")x)"
+    sqrtsd["pd"] = _mm "_sqrt_pd((" __m "d)x)"
+    mask_all_zero = \
+    "(_mm_movemask_epi32(_mm_cmpeq_epi32((__m128i)mask, " \
+    "_mm_xor_si128((__m128i)mask,(__m128i)mask))) == 15)"
+
```

Comment:
This is adding new code to the Linux AArch64 build. While libpgmath seems to build fine with this addition, it is not clear what this is doing, and whether we have enough test coverage to be sure that this change doesn't break something on Linux. Can you change the awk script to run this block only for the Windows build?